apps/innbyggers-flate/src/components/HvaErDette.tsx

```diff
@@ -0,0 +1,37 @@
+import { BodyLong, Detail, Heading } from '@navikt/ds-react'
+import { formatDateStrWithMonthName } from '../utils/utils.ts'
+import { Vedtaksinformasjon } from '../api/data/deltaker.ts'
+
+interface Props {
+  vedtaksinformasjon: Vedtaksinformasjon | null
+  className: string
+}
+
+export const HvaErDette = ({ vedtaksinformasjon, className }: Props) => {
+  const harNavMeldtPaDirekte = vedtaksinformasjon?.fattetAvNav
+
+  return (
+    <div className={className}>
+      <Heading level="2" size="medium">
+        Hva er dette?
+      </Heading>
+      <BodyLong size="small">
+        Dette er et vedtak etter arbeidsmarkedsloven § 12 og forskrift om
+        arbeidsmarkedstiltak kapittel 4.
```

Comment:
Har du lyst til å hente den oppdaterte `HvaErDette` fra nav-veileder-flaten? 

Vi har vel ikke noe satt opp for å dele ting i mellom appene enda?